app/controllers/AuthenticationController.scala

```diff
@@ -234,9 +234,9 @@ class AuthenticationController @Inject()(
       organization <- organizationDAO.findOne(invite._organization)(GlobalAccessContext) ?~> "invite.invalidToken"
       _ <- userService.assertNotInOrgaYet(request.identity._multiUser, organization._id)
       requestingMultiUser <- multiUserDAO.findOne(request.identity._multiUser)
-      _ <- Fox.runIf(!requestingMultiUser.isSuperUser)(
-        organizationService
-          .assertUsersCanBeAdded(organization._id)(GlobalAccessContext, ec)) ?~> "organization.users.userLimitReached"
+      alreadyPayingOrgaForMultiUser <- userDAO.findPayingOrgaIdForMultiUser(requestingMultiUser._id)
+      _ <- Fox.runIf(!(requestingMultiUser.isSuperUser || alreadyPayingOrgaForMultiUser.isDefined))(organizationService
```

Comment:
maybe resolve the negative and parentheses here?

`runIf(!requestingUser.isSuperUser && alreadyPayingOrgaForMultiUser.isEmpty)`